### PR TITLE
docs(mass-spring-damper-tutorial): update

### PR DESCRIPTION
**Pull request by iftahnaf**
hi @mariaBer,

I update the mass-spring-damper tutorial.

Please review.

### CODE REVIEW

File: docs_tutorials/mass_spring_damper/mass_spring_damper.md
Comment (mariaBer) [inline docs_tutorials/mass_spring_damper/mass_spring_damper.md:1]: Please review requested drone fixes, add then here as well. https://github.com/lulav/citros_doc/pull/76#pullrequestreview-1737421079

File: docs_tutorials/mass_spring_damper/mass_spring_damper.md [hunk 1]
```diff
@@ -55,14 +42,28 @@ $$
...
+```
+Then open the repository in VSCode's `devcontainer` with `reopen in container option`.  
 
-### Configuring The Project
+The Dockerfile contains all the necessary dependencies for the project.
 
-After all the prerequisites done, we can start configuring our project. Open the project's repository in `VSCode` and reopen the project in a the VSCode's `devcontainer`.
+## Workspace Overview
 
 ### Parameters
```
Comment (mariaBer) [inline docs_tutorials/mass_spring_damper/mass_spring_damper.md:69]: Please change to composition of the fields to look like in drone (or update in drone to look like this)

| Parameter | Description | Package

File: docs_tutorials/mass_spring_damper/mass_spring_damper.md [hunk 2]
```diff
@@ -86,24 +87,10 @@ After all the prerequisites done, we can start configuring our project. Open the
...
+Make sure to install and initialize CITROS by following the [Getting Started](https://citros.io/doc/docs_tutorials/getting_started/) tutorial.
 
-Follow the [guide](https://citros.io/doc/docs/cli/cli_commands/cli_sync) for syncing CITROS project with the server.
-
-
-
-### Normal Distributed Mass
+## Scenario
 
 Supposed we tune the PID gains of the controller for the nominal mass, and we reached a satisfying results.
```
Comment (mariaBer) [inline docs_tutorials/mass_spring_damper/mass_spring_damper.md:96]: line 116 (https://citros.io/doc/docs/repos/repos_file_structure/repos_fs_param_setup). not exist

**Review by mariaBer — CHANGES_REQUESTED** [review]

**Comment by iftahnaf** [comment]
Hi @mariaBer, I fixes the issues, please review.

Thanks.

File: docs_tutorials/mass_spring_damper/mass_spring_damper.md [hunk 3]
```diff
@@ -4,28 +4,17 @@ sidebar_label: 'Mass Spring Damper'
 
 ---
 
-# Mass Spring Damper
+## Mass-Spring-Damper
```
Comment (mariaBer) [inline docs_tutorials/mass_spring_damper/mass_spring_damper.md:7]: # Mass-Spring-Damper

File: docs_tutorials/mass_spring_damper/mass_spring_damper.md [hunk 4]
```diff
@@ -55,29 +44,43 @@ $$
...
+For working without dockers **(not recommended)**, please check the [.devcontainer](https://github.com/citros-garden/mass-spring-damper/tree/main/.devcontainer) folder in the project's repo for dependencies (in the `Dockerfile` and `install.sh`).
 
-### CITROS Installation
-First of all, to use all the powerful CITROS features usage requires CITROS installation: follow the instructions on the CITROS CLI [documentation](https://citros.io/doc/docs_cli/)
+## Table of Contents
+1. [Installation](#installation)
+2. [Workspace Overview](#workspace-overview)
+3. [CITROS Initialization](#citros-initialization)
+4. [Scenario](#scenario)
+5. [Running the scenario using CITROS](#running-the-scenario-using-citros)
```
Comment (mariaBer) [inline docs_tutorials/mass_spring_damper/mass_spring_damper.md:56]: Capitalise  - Running the Scenario Using CITROS

File: docs_tutorials/mass_spring_damper/mass_spring_damper.md [hunk 5]
```diff
@@ -139,6 +128,15 @@ We will define the following requirements:
 * Settling time is `2.0` [sec].
 * Settling to `10%` of the steady-state value.
 
+## Running the scenario using CITROS
```
Comment (mariaBer) [inline docs_tutorials/mass_spring_damper/mass_spring_damper.md:131]: Capitalise  - Running the Scenario Using CITROS

File: docs_tutorials/mass_spring_damper/mass_spring_damper.md [hunk 6]
```diff
@@ -55,29 +44,43 @@ $$
...
+4. [Scenario](#scenario)
+5. [Running the scenario using CITROS](#running-the-scenario-using-citros)
+6. [Results](#results)
+
+## Installation
+Clone the repository from Github:
+```sh
+git clone git@github.com:citros-garden/mass-spring-damper.git
+```
+Then open the repository in VSCode's `devcontainer` with `reopen in container option`.
```
Comment (mariaBer) [inline docs_tutorials/mass_spring_damper/mass_spring_damper.md:64]: `reopen in container` option.

**Review by mariaBer — CHANGES_REQUESTED** [review]

**Comment by mariaBer** [comment]
@iftahnaf, did you see I added a few new comments on the last update?

**Comment by iftahnaf** [comment]
Hi @mariaBer,  I update the tutorial according to your request.
Please review it.

**Review by mariaBer — APPROVED** [review]

**Pull request merged**